main_test.go

```diff
@@ -19,3 +19,15 @@ func Test_needsChangeMessage_returnFalse(t *testing.T) {
 		t.Errorf("got: %v\nwant: %v", actual, expected)
 	}
 }
+
+func Test_pickupSquashRange(t *testing.T) {
+  begin, end := pickupSquashRange("2")
+  if begin != 2 || end != 0 {
+    t.Errorf("begin: %v\nend: %v", begin, end)
+  }
+
+  begin, end = pickupSquashRange("2..4")
```

Comment:
Add the `pickupSquashRange("4..2")` to check that it is enabled whether swapping is.

Reply:
Thanks! addressed https://github.com/kamontia/qs/pull/122/commits/f09718f03b9381f1a101ae0e2247233b9a54fd06